### PR TITLE
Updates from rspec-dev (2020-09-28)

### PR DESCRIPTION
**Pull request by benoittgt**
These are some updates, generated from rspec-dev's rake tasks.

https://github.com/rspec/rspec-dev/pull/250

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,16 +39,12 @@ matrix:
...
-      env: DIFF_LCS_VERSION="1.4.3"
   allow_failures:
     - rvm: jruby-head
     - rvm: ruby-head
     - rvm: rbx-3
   fast_finish: true
 branches:
   only:
-    - main
+    - master
```
Comment (JonRowe) [inline .travis.yml:49]: Whoops?

**Review by JonRowe — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (benoittgt) [reply .travis.yml]: yep :/

File: .travis.yml [hunk 2]
```diff
@@ -40,8 +41,6 @@ matrix:
       env: JRUBY_OPTS='--dev --1.8'
     - rvm: 2.7.1
       env: DIFF_LCS_VERSION="~> 1.3.0"
-    - rvm: 2.7.1
-      env: DIFF_LCS_VERSION="1.4.3"
```
Comment (JonRowe) [inline .travis.yml:44]: This needs to be restored before merge.

**Comment by benoittgt** [comment]
Closing for https://github.com/rspec/rspec-support/pull/433/files

**Pull request closed without merging**